src/_macosx.m

```diff
@@ -26,13 +23,6 @@
 #define COMPILING_FOR_10_10
 #endif
 
-/* Use Atsui for Mac OS X 10.4, CoreText for Mac OS X 10.5 */
-#ifndef COMPILING_FOR_10_5
```

Comment:
BTW, I am planning on submitting another related patch to avoid all the deprecation warnings when compiling in 10.12